R/boxr_misc.R

```diff
@@ -27,8 +27,18 @@ box_ls <- function(dir_id = box_getwd(), limit = 100, max = Inf) {
     paste(url_root, "folders", box_id(dir_id), "items", sep = "/")
   )
 
-  fields <- c("modified_at" ,"content_modified_at", "name", "id", "type",
-              "sha1" ,"size", "owned_by", "path_collection", "description")
+  filelds_all <- 
```

Comment:
This was a typo on my part - should be `fields_all`.

Can you build-and-install on your end, just to make sure the function works as you expect?

In the longer term, I will have to look more-closely at the tests.

Thanks!